setup.py

```diff
@@ -13,63 +12,44 @@
           '-Wno-cast-function-type', '-std=c99', '-D_FILE_OFFSET_BITS=64']
 
 alpm = Extension('pyalpm',
-    libraries=['alpm'],
-    extra_compile_args=cflags + ['-DVERSION="%s"' % pyalpm_version],
-    language='C',
-    sources=[
-        'src/pyalpm.c',
-        'src/util.c',
-        'src/package.c',
-        'src/db.c',
-        'src/options.c',
-        'src/handle.c',
-        'src/transaction.c'
-        ],
-    depends=[
-        'src/handle.h',
-        'src/db.h',
-        'src/options.h',
-        'src/package.h',
-        'src/pyalpm.h',
-        'src/util.h',
-        ])
-
-class TestCommand(Command):
-    user_options = []
-
-    def initialize_options(self):
-        pass
-
-    def finalize_options(self):
-        pass
-
-    def run(self):
-        raise SystemExit(
-            subprocess.call(['nosetests',
-                             'tests']))
+                 libraries=['alpm'],
+                 extra_compile_args=cflags + ['-DVERSION="%s"' % pyalpm_version],
+                 language='C',
+                 sources=['src/pyalpm.c',
+                          'src/util.c',
+                          'src/package.c',
+                          'src/db.c',
+                          'src/options.c',
+                          'src/handle.c',
+                          'src/transaction.c'],
+                 depends=['src/handle.h',
+                          'src/db.h',
+                          'src/options.h',
+                          'src/package.h',
+                          'src/pyalpm.h',
+                          'src/util.h'])
 
 
 with open("README", "r") as fh:
     long_description = fh.read()
 
+pycman_scripts = ['database', 'deptest', 'query', 'remove', 'sync', 'upgrade', 'version']
```

Comment:
I wonder if we'd like to name this "variable" in uppercase, so as to make it clear it's a "constant"

Reply:
seems alright :)